src/components/reducer_and_actions.js

```diff
@@ -0,0 +1,67 @@
+import {getBreedImages} from "./api";
+import {getBreeds} from './api';
+
+
+//breed images
+const initialState = [];
+export const imagesReducer = (state = initialState, action) => {
```

Comment:
Separate reducers and actions into their own files, for example ./src/store/reducers/reducerName.js, ./src/store/actions/reducerActionName.js